Gemfile

```diff
@@ -6,6 +6,8 @@ gem 'ruby-fifo', github: 'shurizzle/ruby-fifo'
 
 group :development do
   gem 'rspec'
+  gem 'coveralls'
+  gem 'simplecov'  
```

Comment:
@dmcnulla I meant that to do as here: https://github.com/dmcnulla/ntp/blob/master/ntp-mock-server.gemspec#L30